bigframes/session/__init__.py

```diff
@@ -1970,3 +2016,21 @@ def _transform_read_gbq_configuration(configuration: Optional[dict]) -> dict:
         configuration["jobTimeoutMs"] = timeout_ms
 
     return configuration
+
+
+def _delete_tables_matching_session_id(
```

Comment:
Seeing as this doesn't require any Session object stuff, could we move this to bigframes/session/_io/bigquery.py ? This file is getting a bit long for my editor to handle comfortably sometimes.

Reply:
Roger, moved